expression/column.go

```diff
@@ -297,6 +302,11 @@ func (col *Column) IsCorrelated() bool {
 	return false
 }
 
+// ReferTable implements Expression interface.
+func (col *Column) ReferTable() bool {
+	return true
```

Comment:
If the column is generated by project like `1 as a`. Should it also return true?

Reply:
`1 as a` may create a tmp table, I think it should be true.
```
mysql> select unix_timestamp(tmp) from (select "2017-01-02" as tmp) as c;
+---------------------+
| unix_timestamp(tmp) |
+---------------------+
|   1483286400.000000 |
+---------------------+
1 row in set (0.01 sec)

mysql> select unix_timestamp("2017-01-02");
+------------------------------+
| unix_timestamp("2017-01-02") |
+------------------------------+
|                   1483286400 |
+------------------------------+
1 row in set (0.00 sec)
```